### DISCUSSION TITLE
Weather Forecast

### DISCUSSION THREAD

**Discussion by erte1** [thanksmister/android-mqtt-alarm-panel#202] [Q&A]
My panel has had issues with displaying the weather forecast ever since forecast has been deprecated.  Has anyone figured out the formatting to send weather.get_forecasts to the panel?

**Comment by fireport** [comment]
Dear @erte1 I posted a solution that works for me here: [https://community.home-assistant.io/t/wheather-forecasts-output-compatible-with-android-mqtt-alarm-panel/732081/8?u=fireport](url)

**Reply by BrandonG777** [reply]
404 response
